src/Symfony/Component/Mime/NamedAddress.php

```diff
@@ -39,4 +41,25 @@ public function toString(): string
     {
         return $this->getEncodedNamedAddress();
     }
+
+    public static function fromString(string $string): self
+    {
+        if (false === strpos($string, '<')) {
+            return new static($string, $string);
+        }
+
+        $matched = preg_match(
+            static::FROM_STRING_PATTERN,
+            $string,
+            $matches
+        );
```

Comment:
Should be on one line and could be moved to the condition: `if (!$matched = $matched = preg_match(static::FROM_STRING_PATTERN, $string, $matches) {`